### PR TITLE
End-To-End Dokumentation and cleanup

### PR DESCRIPTION
**Pull request by SuyDesignz**
@jplag/maintainer  @jplag/studdev 
Changes made to README.md documentation. Hereby spelling mistakes, as well as structures, were revised which should provide better comprehensibility and readability. Adapt and normalize expressions to uniformity in the terms. Also change of options for the test runs, because the old options do not provide enough coverage for default settings from (1, 15) to (9, 3).

### CODE REVIEW

File: jplag.endtoend-testing/src/test/java/de/jplag/endtoend/EndToEndSuiteTest.java [hunk 1]
```diff
@@ -60,8 +60,8 @@ public EndToEndSuiteTest() throws IOException {
      */
     private void setRunOptions() {
         options = new ArrayList<>();
-        options.add(new Options(1));
-        options.add(new Options(15));
+        options.add(new Options(9));
+        options.add(new Options(3));
     }
```
Comment (JanWittler) [inline jplag.endtoend-testing/src/test/java/de/jplag/endtoend/EndToEndSuiteTest.java:65]: I still don't understand why this has to be specified in code in addition to the JSON?

Comment (SuyDesignz) [reply jplag.endtoend-testing/src/test/java/de/jplag/endtoend/EndToEndSuiteTest.java:65]: The JSON file is only stored for the results of the tests and not for the creation of test cases. It is not intended to load information from the JSON data to generate test cases. The test cases are generated in the code and the results are stored in the JSON files. This is necessary to be able to recognize the results of the last change in the code.

**Comment by tsaglam** [comment]
@SuyDesignz this is currently still a draft PR. Is that correct?

**Comment by dfuchss** [comment]
@SuyDesignz please resolve merge conflicts. You may have to regenerate the results.

**Review by tsaglam — CHANGES_REQUESTED** [review]
This PR is a bit confusing. @SuyDesignz can you elaborate on why you changed the min token matches?

**Comment by JanWittler** [comment]
> This PR is a bit confusing. @SuyDesignz can you elaborate on why you changed the min token matches?

Probably due to my comments in #632:
> Also, you motivate things using a minimum token match of 1. This value makes no sense at all for code, as this will trigger a match whenever two times the same token is extracted and does not require any structural similarity. Also, why 15? The default match of Java is 9 and thus this should be the first reference to use. As a comparison, you could e.g. take 3,4,5 or similar. They should still show the point that a lower MTM increases the similarity.

**Review by dfuchss — APPROVED** [review]

**Review by tsaglam — APPROVED** [review]

**Pull request merged**